.docker/spark-defaults.conf

```diff
@@ -0,0 +1,17 @@
+# Default system properties included when running spark-submit.
+# This is useful for setting default environmental settings.
+
+# Example:
+# spark.master                     spark://master:7077
+# spark.eventLog.enabled           true
+# spark.eventLog.dir               hdfs://namenode:8021/directory
+# spark.serializer                 org.apache.spark.serializer.KryoSerializer
+# spark.driver.memory               16g
+# spark.executor.extraJavaOptions  -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
+
+spark.jars.packages org.apache.hadoop:hadoop-azure:3.3.2,org.apache.hadoop:hadoop-common:3.3.2,io.delta:delta-core_2.12:2.3.0
+
+# spark.hadoop.fs.AbstractFileSystem.abfss.impl org.apache.hadoop.fs.azurebfs.Abfss
+# spark.hadoop.fs.abfss.impl org.apache.hadoop.fs.azurebfs.SecureAzureBlobFileSystem
```

Comment:
Do we need to have a reference to these if they are uncommented anyways? Is it to make local development or soemthing easier as the comment above?

Reply:
No idea. I just copy-pasted it, but good point. I will remove it and see :D